internal/kafka/internal/services/clusters.go

```diff
@@ -415,7 +414,7 @@ func (c clusterService) FindKafkaInstanceCount(clusterIDs []string) ([]ResKafkaI
 	for _, k := range kafkas {
 		kafkaInstanceSize, e := c.kafkaConfig.GetKafkaInstanceSize(k.InstanceType, k.SizeId)
 		if e != nil {
-			return nil, apiErrors.NewWithCause(apiErrors.ErrorInstancePlanNotSupported, e, "failed to query kafkas")
+			return nil, e
```

Comment:
Didn't see the need to wrap this error as this method returns an already descriptive error message. It returns one of the following error message:
- `Unable to find kafka instance type for '<instance-type>'`
- `Kafka instance size id: '<size-id>' not found for '<region>' instance type`.

The caller of this method can wrap the error returned by this instead.